packages/manager/apps/pci-instances/src/routes/routes.tsx

```diff
@@ -104,6 +104,13 @@ const instanceLegacyRoutes: RouteObject[] = instanceActionsSections.map(
   }),
 );
 
+const instanceActionRoutes = instanceActionsSections.map((section) => ({
```

Comment:
just a detail for instancesActionRoutes & instanceActionRoutes, I think both should be plural (Action => Actions).
As instanceActionsSections is already plural